tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -102,9 +118,9 @@ def example_plot(ax, fontsize=12, nodec=False):
 #
 # .. note::
 #
-#   For the `pcolormesh` kwargs (``pc_kwargs``) we use a dictionary.
+#   For the `~axes.Axes.pcolormesh` kwargs (``pc_kwargs``) we use a dictionary.
```

Comment:
`~.Axes.pcolormesh` (otherwise it would have to be `~axes._axes.Axes.pcolormesh`)

Reply:
This is the error I make most often with this kind of format. I always forget the leading dot.

`~.axes.Axes.pcolormesh` works fine. I've been using this quite often in recent PRs. This is because `axes/__init__.py` has `from ._axes import *` in it. Hence `matplotlib.axes.Axes.pcolormesh` is valid.